### DISCUSSION TITLE
apparently -very non important- bug in api root

### DISCUSSION THREAD

**Discussion by joseespinadote** [encode/django-rest-framework#8279] [Potential Issue]
(newbie on DRF, and non english speaker here, sorry beforehand)
Hi! I created two ModelViewSet classes in my views.py for the same model (may be that's not the best practice), because I needed two totally different serializers to acomplish some frontend requirements
The generated list, at the api root site, duplicates one of the entries (just the url). See the picture for details
![imagen](https://user-images.githubusercontent.com/2145749/144780828-a3157ff3-bfe2-46d1-ac07-e52b668f712c.png)
This doesnt' affect the functionality at all. But as I'm DRF user, I wanted to support notifying this concern
Bests, and thanks for creating DRF

PS.: this is part of the code at the urls.py:
```
router = routers.DefaultRouter()
router.register(r'accion', views.AccionViewSet)
router.register(r'entregable', views.EntregableViewSet)
router.register(r'_entregable', views.EntregableDepthViewSet)
router.register(r'retroEntregable', views.RetroEntregableViewSet)
router.register(r'adjuntos', views.AdjuntoEntregableViewSet)
```

**Comment by s4ke** [comment]
I dont see any duplication. You are registering entregable and _entregable. Both are different viewsets.

Maybe share your two viewsets, then you can get better help :)

**Comment by joseespinadote** [comment]
The url is duplicated

**Reply by lovelydinosaur** [reply]
Yes that looks a bit odd, doesn't it. Perhaps there's some kind of conflict between the naming of `entregable` and `_entregable` that's causing the two different cases both to map to `/api/_entregable`.

You might want to dig in and see if that replicates in other cases where you've got two prefixes that are identical except for a leading underscore. And perhaps just change the naming?

**Reply by joseespinadote** [reply]
Thanks @tomchristie . Good idea
It's not the prefix. I changed '_entregable' for 'entregable2' in urls.py, and look:
![imagen](https://user-images.githubusercontent.com/2145749/145270463-62b08e88-266d-4f7b-89b2-5c00f3ef3849.png)

This is part of my views.py that I forgot to share in the first post:
```
class EntregableViewSet(viewsets.ModelViewSet):
    queryset = Entregable.objects.all().order_by('-id')
    serializer_class = EntregableSerializer
    permission_classes = [permissions.IsAuthenticated]
    filter_backends = [DjangoFilterBackend]
    filterset_fields = ['usuario', 'mdv', 'nombre', 'estado']

class EntregableDepthViewSet(viewsets.ModelViewSet):
    queryset = Entregable.objects.all().order_by('-id')
    serializer_class = EntregableDepthSerializer
    permission_classes = [permissions.IsAuthenticated]

   # ...and some custom actions are here
```
